framework/components/ecl-site-switchers/ecl-site-switchers.twig

```diff
@@ -1,6 +1,12 @@
-<div class="site-switcher {{modifier}}">
-  <ul class="site-switcher__list">
-      <li class="site-switcher__option"><a href="#">Commission and its priorities</a></li>
-      <li class="site-switcher__option is-selected"><a href="#">Policies, information and services</a></li>
+{#
+  - modifier: can be '','header','footer' (default: '')
+  - political: label (default: '')
+  - info: label (default: '')
+#}
+
```

Comment:
Please use the same structure for twig file as in other recent files:
- api interface
- variable declaration and initialisation
- markup

See featured-items for instance